src/test/java/com/talentpath/airTNB/services/ActivityTests.java

```diff
@@ -0,0 +1,138 @@
+package com.talentpath.airTNB.services;
+
+import com.talentpath.airTNB.daos.ActivityRepository;
+import com.talentpath.airTNB.models.Activity;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+@ExtendWith(SpringExtension.class)
+public class ActivityTests {
+
+    @Autowired
+    ActivityRepository activityRepo;
+
+    @BeforeEach
+    @Transactional
+    void setup(){
+        activityRepo.deleteAll();
+        activityRepo.reset();
+        Activity activity = new Activity();
+        activity.setCity("Houston");
+        activity.setRegion("Texas");
+        activityRepo.saveAndFlush(activity);
+    }
+
+    @Test
+    @Transactional
+    void getAllActivities(){
+        List<Activity> activities = activityRepo.findAll();
+        assertEquals(1, activities.size());
+
+        Activity politicsTour = new Activity();
+        politicsTour.setCity("Washington DC");
+        politicsTour.setPrice(75);
+        activityRepo.saveAndFlush(politicsTour);
+
+        Activity tourFacebook = new Activity();
+        tourFacebook.setCity("Menlo Park");
+        tourFacebook.setPrice(30);
+        activityRepo.saveAndFlush(tourFacebook);
+
+        activities = activityRepo.findAll();
+        assertEquals(3, activities.size());
+        assertEquals("Washington DC", activities.get(1).getCity());
+        assertEquals(75, activities.get(1).getPrice());
+        assertEquals("Menlo Park", activities.get(2).getCity());
+        assertEquals(30, activities.get(2).getPrice());
+    }
+
+    @Test
+    @Transactional
+    void getActivityById(){
```

Comment:
This is only testing findall and the auto-increment of the database, needs something along the lines of 
"activityrepository.find('primary_key')" to test find by id

Reply:
I'm a little confused as to what the difference would be between find(primary_key) and findById(id) since the id for this model is the primary key.  So when I want to look up a specific activity, I can send the id into the Jpa method and it will select the exact activity I need

Reply:
yes findbyid! my bad. I messed up on the method name.

In this test theres only findAll()

Reply:
Oh wow yup you're right let me add that in